### ISSUE TITLE
New cookies dialog is not accessible in dark mode: contrast

### ISSUE THREAD

**Issue by Splaktar** [angular/material.angular.io#1008]
# Purple and Green

![Screen Shot 2021-06-06 at 20 02 17](https://user-images.githubusercontent.com/3506071/120944412-58e1ff00-c702-11eb-9eec-ad6d05c0d5c8.png)
![Screen Shot 2021-06-06 at 20 02 40](https://user-images.githubusercontent.com/3506071/120944411-58496880-c702-11eb-85d5-cf8ca12d84ec.png)

# Pink and Blue-grey

![Screen Shot 2021-06-06 at 20 03 06](https://user-images.githubusercontent.com/3506071/120944410-57b0d200-c702-11eb-9371-c342ce7f522f.png)
